packages/manager/modules/pci/src/components/order-command/order-command.html

```diff
@@ -0,0 +1,30 @@
+<h5 data-translate="pci_order_command_title"></h5>
+<div>
+    <ng-transclude></ng-transclude>
+</div>
+<tail-logs data-ng-class="{'overloader_height_tail_logs': $ctrl.autoHeight }">
+    <p data-ng-bind="$ctrl.orderApiUrl"></p>
+    <p data-translate="pci_order_command_parameters"></p>
+    <pre data-ng-bind="$ctrl.getRequestData()"></pre>
+    <p>
+        <span data-translate="pci_order_command_parameters_info"></span>
+        <a data-ng-if="$ctrl.helpLink" data-ng-href="{{:: $ctrl.helpLink}}">
+            <span data-translate="pci_order_command_here"></span>
```

Comment:
**nitpick:** `data-translate` directive can be applied directly to the `<a>` element.

Reply:
Done